toolchain/lower/testdata/global/use.carbon

```diff
@@ -0,0 +1,53 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/lower/testdata/global/use.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/lower/testdata/global/use.carbon
+var a: i32;
+
+var b: i32 = a;
```

Comment:
I would imagine a future version of the compiler rejecting this code, for using a variable in unformed state. Better to keep that out of a test that isn't trying to test that thing.

Reply:
Ah, I wanted to test the use of the global from another global. If I initialized `a` with some value, would that be acceptable/less likely to be made invalid in the future?

Reply:
Yes

Reply:
Done in 150beab9fa18b199bfe27061c64e2225135b01b4